en/docs/install-and-setup/setup/setting-up-databases/managing-data-growth-and-improving-performance.md

```diff
@@ -126,3 +126,268 @@ This will remove the old and invalid tokens, sessions, and Auth codes, which can
     ```
 
 Replace `WSO2AM_DB` with the name of your WSO2 API Manager database in the above script.
+
+## Registry database cleanup
+
+Creating and updating APIs, adding tags and ratings cause different registry properties to be added to the Registry database(Registry DB) in WSO2 API-M. When `Registry Versioning` is enabled, older properties are persisted in the database leading to unnecessary record growth in the Registry tables in the database. This directly affects the performance of the product if the number of records increase beyond a certain amount. Due to this limitation, registry versioning is disabled by default in WSO2 API Manager 3.0.0 onwards to prevent unnecessary database growth.
```

Comment:
```suggestion
Creating and updating APIs and adding tags and ratings cause different registry properties to be added to the Registry database (Registry DB) in WSO2 API-M. When `Registry Versioning` is enabled, older properties are persisted in the database leading to unnecessary record growth in the Registry tables in the database. This directly affects the performance of the product if the number of records increase beyond a certain amount. Due to this limitation, registry versioning is disabled by default in WSO2 API Manager 3.0.0 onwards to prevent unnecessary database growth.
```